config/dictionaries/device.json

```diff
@@ -1,389 +1,1361 @@
+      },
+      {
+        "i": "<strong role=\"alert\">Warning!</strong> This will also completely erase internal storage media, so please make a backup first."
+      },
+      {
+        "i": "The device is performing a shutdown. Please wait until all LEDs are off."
+      },
+      {
+        "i": "Remove the balenaOS installation media."
+      },
+      {
+        "i": "Set the BOOT_SELECT switch to the eMMC position."
+      },
+      {
+        "i": "Remove and re-connect power to the board."
+      }
+    ]
   },
   {
     "id": "intel-edison",
     "name": "Intel Edison",
     "arch": "i386",
-    "bootMedia": "eMMC",
+    "bootMedia": "SD card",
+    "state": "RELEASED",
+    "machine": "edison",
+    "community": false,
     "icon": "/img/device/intel-edison.svg",
-    "link": "https://software.intel.com/en-us/iot/hardware/edison/documentation",
-    "type": "released"
+    "instructions": [{
+        "i": "windows"
+      },
+      {
+        "i": "osx"
+      },
+      {
+        "i": "linux"
+      }
+    ]
   },
   {
     "id": "intel-nuc",
     "name": "Intel NUC",
-    "variants": [
+    "arch": "amd64",
+    "bootMedia": "USB key",
+    "state": "RELEASED",
+    "machine": "genericx86-64",
+    "community": false,
+    "icon": "/img/device/intel-nuc.svg",
+    "instructions": [{
+        "i": "Write the OS file you downloaded to your USB key. We recommend using <a href=\"http://www.etcher.io/\">Etcher</a>."
+      },
       {
-        "name": "Intel NUC",
-        "image": "/img/intel-nuc/intel-nuc.jpg",
-        "link": "https://www.intel.com/content/www/us/en/products/boards-kits/nuc.html"
+        "i": "Insert the freshly burnt USB key into the Intel NUC."
       },
       {
-        "name": "Generic x86 devices",
-        "image": "/img/intel-nuc/x86.png",
-        "link": "/learn/getting-started/intel-nuc/nodejs/"
+        "i": "<strong role=\"alert\">Warning!</strong> This will also completely erase internal storage media, so please make a backup first."
+      },
+      {
+        "i": "Please make sure you do not have any other USB keys inserted.\nPower up the Intel NUC. Make sure you have a keyboard connected.\nPress the F10 key (may differ on some platforms) while BIOS is loading in order to enter the boot menu.\nNext, select the name of your USB key."
+      },
+      {
+        "i": "The device is performing a shutdown. Please wait until all LEDs are off."
+      },
+      {
+        "i": "Remove the balenaOS installation media."
+      },
+      {
+        "i": "Power on your device."
       }
-    ],
+    ]
+  },
+  {
+    "id": "surface-pro-6",
+    "name": "Microsoft Surface 6",
     "arch": "amd64",
-    "bootMedia": "USB drive",
-    "icon": "/img/device/intel-nuc.svg",
-    "link": "https://www.intel.com/content/www/us/en/products/boards-kits/nuc.html",
-    "type": "released"
+    "bootMedia": "USB key",
+    "state": "EXPERIMENTAL",
```

Comment:
i'm sure this is pulled from somewhere else, but i thought these were changed to `NEW`? just curious how it fits together

Reply:
So the API returns this and then it seems like it is down to each individual implementation to format it. The docs do this here: https://github.com/balena-io/docs/blob/master/templates/_devices.html#L22 but I've seen similar code in the dashboard (and I think SDK). Either way, I'll update this config file to use the updated ones, as though it isn't used anywhere currently and I left in to minimize changes I could see us using that in the future.

Reply:
Oh, new boards default to `NEW` so I think it's just old ones.